replacement_files/config_structure.cpp

```diff
@@ -2277,6 +2277,18 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   /* DESCRIPTION:  preCICE configuration file name */
   addStringOption("PRECICE_CONFIG_FILENAME", preciceConfigFileName, string("precice.xml"));
```

Comment:
Maybe also update this with `precice-config.xml` as default?

Reply:
Done